doc/rst/source/grdmix.rst

```diff
@@ -0,0 +1,177 @@
+.. index:: ! grdmix
+.. include:: module_core_purpose.rst_
+
+******
+grdmix
+******
+
+|grdmix_purpose|
+
+Synopsis
+--------
+
+.. include:: common_SYN_OPTs.rst_
+
+**gmt grdmix**
+*raster1* [ *raster2* [ *raster3*]]
+|-G|\ *outfile*
+[ |-A|\ *alpha* ]
+[ |-C| ]
+[ |-D| ]
+[ |-I|\ *intensity* ]
+[ |-M| ]
+[ |-N|\ [*factor*] ]
+[ |-Q| ]
+[ |SYN_OPT-R| ]
+[ |SYN_OPT-V| ]
+[ |-W|\ *weights* ]
+[ |SYN_OPT-f| ]
+[ |SYN_OPT--| ]
+
+|No-spaces|
+
+Description
+-----------
+
+**grdmix** will perform various operations involving images and grids.
+We either use a *alpha* grid, image, or constant to add a new alpha
+(transparency) layer to the image given as *raster1*, or we will blend
+the two *raster1* and *raster2* (grids or images) using the *weights* for
+*raster1* and the complementary *1 - weights* for *raster2* and save to
+*outfile*. Alternatively, we will deconstruct an image into its component
+grid layers or we construct an image from its normalized component grids.
```

Comment:
The meaning of "component grid layers" is not clear. Perhaps adding "RGB" is better?
```suggestion
*outfile*. Alternatively, we will deconstruct an image into its RGB component
grid layers or we construct an image from its normalized RGB component grids.
```

Reply:
Done